### DISCUSSION TITLE
registry of community Resources?

### DISCUSSION THREAD

**Discussion by nrvale0** [concourse/concourse#7477] [Help & Support]
Beyond the Resources shipped and supported in the Concourse core, is there a registry of Resources/Resource Types a la Puppet Module Forge, Chef Supermarket, etc etc?

For example, I'm hoping to find a Packer build resource. There are a handful on GH but perhaps there's a closer-to-canonical index versus searching GH?

**Comment by maestrow** [comment]
Hello. There is community resource types registry: https://resource-types.concourse-ci.org/. Or on github: https://github.com/concourse/resource-types. Also see how to install third party resource types to worker: https://github.com/concourse/concourse/discussions/6267

**Comment by nrvale0** [comment]
Thank you! That's helpful. Unfortunately it looks like the Packer ones are not listed but it'll be a good reference for the future.
